static/js/components/LearnerSearch.js

```diff
@@ -220,7 +226,7 @@ export default class LearnerSearch extends SearchkitComponent {
             title="Final Grade in Selected Course"
```

Comment:
Those filters hide automatically, I had to add this code here explicitly because Searchkit doesn't hide this filter automatically.